Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +181,7 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
+        eventDetailViewController.title = event.title;
```

Comment:
Let's set the title in the view controller's `viewDidLoad` instead of in this method. What do you think?